rediscluster/connection.py

```diff
@@ -66,7 +66,7 @@ class ClusterConnectionPool(ConnectionPool):
     RedisClusterDefaultTimeout = None
 
     def __init__(self, startup_nodes=None, init_slot_cache=True, connection_class=ClusterConnection,
-                 max_connections=None, max_connections_per_node=False, **connection_kwargs):
+                 max_connections=None, max_connections_per_node=False, pub_sub_pool_size = 500, **connection_kwargs):
```

Comment:
Pep8 violation, should be `pub_sub_pool_size=500` see the other options on same row.